utils/hare-status

```diff
@@ -304,6 +320,10 @@ def main(argv=None):
         print('Cluster is not running', file=sys.stderr)
         return 1
 
+    if opts.devices:
+        global devicesRequested
```

Comment:
global flag is not the best solution

Reply:
Note: camelCase is not very common for Python. 
Suggestion: `s/devicesRequested/devices_requested/g`

Reply:
Done